tools/clang/test/CodeGenSPIRV/intrinsics.get-attribute-at-vertex.hlsl

```diff
@@ -1,25 +1,30 @@
 // RUN: %dxc -T ps_6_1 -E main
 
-struct PSInput {
-  float4 position : SV_POSITION;
-  nointerpolation float3 color : COLOR;
```

Comment:
Can you also include a test making sure that having inputs in a struct still works?

Reply:
I'll have a try.